### PR TITLE
Feature/admin dashboard - header, sidebar-nav and footer added

### PR DESCRIPTION
**Pull request by Kompweb**
# Description:
1. Update and create new header elements
2. Created new side-bar navigation
3. Created footer 

[X ] My changes generate new warnings:
Warning: Encountered two children with the same key, `link`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

## Fixes #

- What work was done?
The admin dashboard contains AntD components to be consistent with the application. 
- Why was this work done?
produce a better design on the admin dashboard.
- What feature / user story is it for?
- Admin/dashboard
- Any relevant links or images / screenshots
Trello card link: [(https://trello.com/c/Npb2pDTY)]

## Type of change
Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

## Change Status

- [ x] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ x] Yes

## Checklist

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

Screen Shot 2022-05-27 at 3 03 43 PM

Screen Shot 2022-05-27 at 2 59 58 PM

### CODE REVIEW

**Comment by Kompweb** [comment]
The Bug/Error issue noted above has been fixed and the branch updated.
The error was given for duplicate "keys" in the sidebar nav menu.

**Review by caitlinkermen — APPROVED** [review]
Looks good

**Comment by chasedonovan** [comment]
Very clean sidebar and footer!

**Review by ashtilawat — APPROVED** [review]

File: src/styles/less/Admin/Header.less [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
-  height: 15vh;
+  height: 10vh;
 }
-
-img {
-  width: 15vh;
+@media @mobile {
+  min-height: 2vh;
+}
+@media @mobileLarge {
```
Comment (jthernandez999) [inline src/styles/less/Admin/Header.less]: Nice use of media queries, especially taking larger mobile devices into account.

**Pull request merged**